src/mudlet-lua/lua/Other.lua

```diff
@@ -248,46 +242,43 @@ end
 ---
 --- @see loadVars
 function remember(varName)
-	if not _saveTable then
-		_saveTable = {}
-	end
-    _saveTable[varName] = _G[varName]
+  if not _saveTable then
+    _saveTable = {}
+  end
+  _saveTable[varName] = _G[varName]
 end
 
 
-
 --- This function should be primarily used by Mudlet. It loads saved settings in from the Mudlet home directory
 --- and unpacks them into the global namespace.
 ---
 --- @see remember
 function loadVars()
-	if string.char(getMudletHomeDir():byte()) == "/" then _sep = "/" else  _sep = "\\" end
-	local l_SettingsFile = getMudletHomeDir() .. _sep .. "SavedVariables.lua"
-	local lt_VariableHolder = {}
-	if (io.exists(l_SettingsFile)) then
-		table.load(l_SettingsFile, lt_VariableHolder)
-		for k,v in pairs(lt_VariableHolder) do
-				_G[k] = v
-		end
-	end
+  local _sep = string.char(getMudletHomeDir():byte()) == "/" and "/" or "\\"
```

Comment:
This seems to be a sensible change, but again does not belong into the context of `timeframe`.